### PR TITLE
smarter grit

### PR DESCRIPTION
**Pull request by tylerrasor**
build a naive bayesian classifier to determine grittiness

### CODE REVIEW

**Review by zendern — APPROVED** [review]
CYDN

**Comment by zendern** [comment]
@tylerrasor gotta run `go fmt` :)

File: main_test.go [hunk 1]
```diff
@@ -27,14 +27,3 @@ func Test_hasAChickenWord(t *testing.T) {
 		t.Errorf("chicken is a chicken")
 	}
 }
-
-func Test_hasAGrittyWord(t *testing.T) {
-
-	if hasAGrittyWord(strset.New("moist")) {
-		t.Errorf("moist is not gritty")
```
Comment (fitzoh) [inline main_test.go:34]: I think we still need to validate that `moist` isn't gritty, unless @DotNetGeekette can confirm grittiness.

**Comment by fitzoh** [comment]
>@tylerrasor gotta run `go fmt` :)

Did it for him in https://github.com/fitzoh/jamie-chicken/pull/3/commits/4f0c8fdd749d93c55a1846ebf8dc2b5acd8ad2de

**Review by zendern — APPROVED** [review]
(☞ ͡° ͜ʖ ͡°)☞

File: main_test.go (hunk 1 above)
Comment (tylerrasor) [reply main_test.go:34]: https://github.com/fitzoh/jamie-chicken/blob/c23269adbf936081951518ca840951187cabf8cf/sentiment_test.go#L47-L53

**Comment by tylerrasor** [comment]
> > @tylerrasor gotta run `go fmt` :)
> 
> Did it for him in [4f0c8fd](https://github.com/fitzoh/jamie-chicken/commit/4f0c8fdd749d93c55a1846ebf8dc2b5acd8ad2de)

Thanks @fitzoh, my format was bad and I should feel bad

**Review by fitzoh — APPROVED** [review]
:shipit:

**Pull request merged**